src/SubtitleOctopus.cpp

```diff
@@ -5,7 +5,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdarg.h>
-#include <string.h>
+#include <string>
```

Comment:
`<string.h>` should still be pulled in here; we use e.g. `memcmp`. C++ `<string>` is separate and is not required to pull in `<string.h>`. If we want to use all C++ names, we could use `<cstring>`, but that's out of scope for this PR.